src/main/kotlin/uk/gov/dluhc/printapi/config/ExplainerPdfTemplateProperties.kt

```diff
@@ -5,15 +5,16 @@ import org.springframework.boot.context.properties.ConstructorBinding
 
 @ConfigurationProperties(prefix = "temporary-certificate.explainer-pdf")
 @ConstructorBinding
-class TemporaryCertificateExplainerPdfTemplateProperties(english: English, welsh: Welsh) :
-    ExplainerPdfTemplateProperties(english, welsh)
+class TemporaryCertificateExplainerPdfTemplateProperties(filenamePrefix: String, english: English, welsh: Welsh) :
```

Comment:
The filename prefix for the generated explainer document is now in the relevant section in `application.yml` (see below). This is because previously the implementation had the filename prefix hardcoded as `temporary-certificate-explainer-document`

Reply:
`TemporaryCertificateExplainerPdfTemplateProperties(filenamePrefix` - I think this is what made me think it was prefix for the template (see later comment regarding changing this property name)